label.js

```diff
@@ -13,33 +13,52 @@ const labelsToRemove = core
 
 async function label() {
   const myToken = core.getInput("repo-token");
+  const ignoreIfAssigned = core.getInput("ignore-if-assigned");
   const octokit = new github.GitHub(myToken);
   const context = github.context;
+  const repoName = context.payload.repository.name;
+  const ownerName = context.payload.repository.owner.login;
+  const issueNumber = context.payload.issue.number;
 
-  let labels = context.payload.issue.labels.map(label => label.name);
+  // query for the most recent information about the issue. Between the issue being created and
+  // the action running, labels or asignees could have been added
+  var updatedIssueInformation = await octokit.issues.get({
+    owner: ownerName,
+    repo: repoName,
+    issue_number: issueNumber
+  });
+
+  if (typeof ignoreIfAssigned !== undefined && ignoreIfAssigned == "True") {
```

Comment:
Is there anyway to make this a boolean in the YAML and then its cast as a boolean here? Just being able to say `ignoreIfAssigned` because its `true` feels cleaner than checking for string match.

Reply:
I changed `ignoreIfAssigned` to me a required input variable and then simplified the whole thing to just `if (ignoreIfAsigned)`. Making it a required makes it safe to not check for undefined